redeployContract.js

```diff
@@ -211,31 +225,54 @@ const deployLargeContract = async (web3, target_address, contract_code, contract
         .on('receipt', function (receipt) {
             initContractAddress = receipt.contractAddress;
             console.log('Init address: ', receipt.contractAddress); // contains the new contract address
+            console.log(`Actual gas used: ${receipt.gasUsed}`);
             return receipt;
+        })
+        .catch(error => {
+            console.log(error);
+            process.exit();
         });
 
     initInstance.options.address = initContractAddress;
+
     // Set storage
+    console.log('Setting storage.');
     const storageKeys = Object.keys(contract_state);
+    let keysPerBatch = 2;
```

Comment:
A descriptive comment may be helpful here ;)

Reply:
will do!